onset_maproom/config-anacim-senegal.yaml

```diff
@@ -19,12 +32,23 @@ rr_mrg_zarr_path: /data/remic/mydatafiles/ANACIM/anacim_v3/ALL_block2/rr_mrg_dai
 logo: anacimLogo.jpg
 institution: ANACIM
 zoom: 7
+shapes_adm1: select id_1 as key, name_1 as label, ST_AsBinary(the_geom) as the_geom
+                  from sen_adm1
+name_adm1: Régions
+shapes_adm2: select id_2 as key, name_2 as label, ST_AsBinary(the_geom) as the_geom
+                  from sen_adm2
+name_adm2: Départements
+shapes_adm3: select id_3 as key, name_3 as label, ST_AsBinary(the_geom) as the_geom
+                  from sen_adm3
+name_adm3: Arrondissements
 
 # Climate and Ag
 
 # Onset_and_Cessation
-onset_and_cessation_title: When to prepare land and inputs for planting?
+app_title: Planting Maproom
+onset_and_cessation_title: Planting Date Decision Support Maproom
 default_search_month: May
 default_running_days: 3
 default_min_rainy_days: 1
 ison_cess_date_hist: False
+and_cess_text: ""
```

Comment:
agreed for ison_cess_date_hist and and_cess_text; but who knows what, say Rwanda, would ask for a titles...
Then I created and_cess_text because I didn't manage to have code with the """ text...

Reply:
I got errors when trying to squeeze an if in the """-string to check boolean and act accordingly. Maybe f-string will solve that?

Reply:
I don't think so. I'd have to see the example to know what you were doing wrong.

In either case, whether you write
```
'abc' + foo + 'def'
```
or 
```
f'abc{foo}def'
```
`foo` has to be an expression, not a statement. Maybe the [ternary operator](https://docs.python.org/3/reference/expressions.html#conditional-expressions) is what you were missing?

Reply:
it's the syntax for if/else showed in the link you shared that I didn't know... I was doing something like 
if boolean: expression